UserManagementService/service/com/taxicoop/service/DBHelper.py

```diff
@@ -1,21 +1,84 @@
+import pprint
+from datetime import datetime, timedelta
+from bson.objectid import ObjectId
+
 import pymongo
+import certifi
+
+
+class Database:
+    DB_NAME = 'user_management'
+
+    # collection ='users'
+
+    def __init__(self):
+        self.db_conn = pymongo.MongoClient(
+            "mongodb+srv://taxicoop:admin123@cluster0.ykco3.mongodb.net/?retryWrites=true&w=majority",
+            tlsCAFile=certifi.where()
```

Comment:
remove this

Reply:
so where would be our connection string ? I know this is your personal account...moving to AWS this would not be needed. But yet we need to connect right ?